utils/R/do.conversions.R

```diff
@@ -1,9 +1,8 @@
-##' @export
+##' @export do.conversions
```

Comment:
OK, we can deal with that later. It definitely does need the **@export** and at some point soon we should switch the name to do_conversions but that doesn't have to be this PR